pkg/api/goprobe/server/server.go

```diff
@@ -25,11 +25,11 @@ func (server *Server) SetDBPath(path string) *Server {
 }
 
 // New creates a new goprobe API server
-func New(addr string, captureManager *capture.Manager, opts ...server.Option) *Server {
+func New(serviceName, addr string, captureManager *capture.Manager, opts ...server.Option) *Server {
```

Comment:
Same here, why do we need to hand over the (constant) service name as parameter?

Reply:
What you suggest makes sense. For the `DefaultServer`, we need the service name, but not for the servers created by the individual tools. Will adjust.